packages/opentelemetry-plugin-postgres/opentelemetry-plugin-pg/src/pg.ts

```diff
@@ -37,7 +37,7 @@ export class PostgresPlugin extends BasePlugin<typeof pgTypes> {
   readonly supportedVersions = ['7.*'];
 
   constructor(readonly moduleName: string) {
-    super();
+    super(`opentelemetry.${moduleName}`, '0.2.0');
```

Comment:
If this (`0.2.0`) is a version of instrumentation library, we need some way to update data during each release cycle, maybe we can read this from `package.json` but I remember there is some issue with web code. Looks like we need to create `version-update.ts` script for each package, correct me if I am wrong.

Reply:
I added the version-update script to all packages